addons/account/views/account_move_views.xml

```diff
@@ -738,11 +738,12 @@
                         <field name="commercial_partner_id" invisible="1"/>
                         <field name="bank_partner_id" invisible="1"/>
                         <field name="display_qr_code" invisible="1"/>
+                        <field name="invoice_pdf_report_id" invisible="1"/>
```

Comment:
unnecessary diff?

Reply:
No, there's some mess going on also in `master`... it's for this change here. The Send and Print button should be highlighted and one not depending on this field.
![image](https://github.com/odoo/odoo/assets/1665365/2414c2c0-ff51-4730-8b3f-db42c30e3024)

Reply:
It was `is_move_send` before

Reply:
That makes sense, I was just puzzled because you delete this field a few lines below (at line 745), and add it back here, but it's not a big deal